pkg/engine/deployment.go

```diff
@@ -8,12 +8,7 @@ import (
 	"github.com/Azure/go-autorest/autorest/to"
 )
 
-const (
-	azurestackGenerateGUID = "pid-1bda96ec-adf4-4eea-bb9a-8462de5475c0"
-)
-
-//TODO create separate function to allow users to pass in deployment name for azure
-func createAzurestackTelemetry() DeploymentARM {
+func createAzurestackTelemetry(azureTelemetryPID string) DeploymentARM {
```

Comment:
Same here. Please change createAzurestackTelemetry to createAzureStackTelemetry

Reply:
done